client/.eslintrc

```diff
@@ -30,6 +30,7 @@
     "max-depth": ["error", 5], // This rule enforces a maximum depth that blocks can be nested.
     // "multiline-comment-style": ["error", "starred-block"], // This rule enforces a particular style for multiline comments.
     "no-else-return": ["error", { "allowElseIf": false }], // This rule disallows unnecessary blocks following if and else statements.
-    "no-empty": ["error", { "allowEmptyCatch": true }] // This rule disallows empty block statements.
+    "no-empty": ["error", { "allowEmptyCatch": true }], // This rule disallows empty block statements.
+    "prettier/prettier": ["error", { "endOfLine": "auto" }] // To remove errors while reloading the server
```

Comment:
I added the same thing in .prettierrc . Was that not working as expected? If not, then I'll accept these.
As you can notice, comments follow a common syntax "This rule ..." try to rewrite it like that.

Reply:
The terminal was throwing errors. When I added this line in .eslintrc file, the errors were gone.
Changed the comment... 891e2f1